client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/index.tsx

```diff
@@ -170,13 +170,28 @@ const PatternAssembler: Step = ( { navigation } ) => {
 		goBack();
 	};
 
+	const getSelectedPattern = () => {
+		if ( 'header' === showPatternSelectorType ) {
```

Comment:
Note for reviewers... I don't use a `switch` here because I don't like them 😝

Reply:
[nit] You could use some tricks as followed 😉

```js
return ({
  header,
  footer,
  'section': sections[ sectionPosition ] ?? null
})[showPatternSelectorType];
```

Reply:
No. Please don't do the above trick. It is unreadable 😢 🙈

Reply:
I don't think it's not readable. We just define an object mapping the pattern type to the selected pattern first, and then get the selected pattern by the pattern type.